james/types.go

```diff
@@ -226,18 +226,22 @@ func (w *WebAdminClient) RenewSession() error {
 	return <-errChan
 }
 
-func (w *WebAdminClient) renew() error {
-	var lastErr error
-	err := wait.PollUntilContextTimeout(
+func (w *WebAdminClient) renew() (err error) {
```

Comment:
Create one renew function to fetch and create the clients.

Reply:
This will take some major refactoring. Should probably do this in another PR.